SICP_book/chapter_1_1.metta

```diff
@@ -0,0 +1,191 @@
+(= (sqr $x) (* $x $x))
+
+!(assertEqual
+    (sqr 21)
+    441)
+
+!(assertEqual
+    (sqr (+ 2 5))
+    49)
+
+!(assertEqual
+    (sqr (sqr 3))
+    81)
+
+(= (sum-of-squares $x $y) (+ (sqr $x) (sqr $y)))
+
+!(assertEqual
+    (sum-of-squares 3 4)
+    25)
+
+(= (f $a) (sum-of-squares (+ $a 1) (* $a 2)))
+
+!(assertEqual
+    (f 5)
+    136)
+
+(= (abs $x)
+    (if (> $x 0)
+        $x
+        (if (== $x 0)
+            0
+            (* $x -1))))
+
+!(assertEqual
+    (abs -5)
+    5)
+
+(= (_abs $x)
+    (if (< $x 0)
+        (* $x -1)
+        $x))
+
+!(assertEqual
+    (_abs -5)
+    5)
+
+(= (>= $x $y) (or (> $x $y) (== $x $y)))
+
+!(assertEqual
+    (>= 6 5)
+    True)
+
+; excercise 1.3
+; Define procedure that takes three numbers as arguments and returns
+; the sum of the squares of the two larger numbers
+
+;I haven't got an idea of function's name
+(= (exercise_1_3 $x1 $x2 $x3)
```

Comment:
This will work only if there will be no equal input numbers

Reply:
you probably right. I haven't checked that. I'll take a look.

Reply:
@mvpeterson  fixed